### PR TITLE
Feature/create new homepage template and route

### PR DESCRIPTION
**Pull request by BarryGeeMOJ**
### Context

> Does this issue have a Trello card?
https://trello.com/c/dJYDWanF/972-create-new-homepage-and-update-existing-homepage-router

> If this is an issue, do we have steps to reproduce?
n/a

### Intent

> What changes are introduced by this PR that correspond to the above card?

- new homepage placeholder view created ready for additional components.
- new route /new-homepage added to enable access to the view during development
- new unit tests added

> Would this PR benefit from screenshots?
no

### Considerations

> Is there any additional information that would help when reviewing this PR?
no

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ x] This PR contains **only** changes related to the above card
- [ x] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [x] Tested in Development

### CODE REVIEW

**Review by leonk — DISMISSED** [review]
LGTM

File: server/views/pages/home-new.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "../components/personal-schedule-today/macro.njk" import personalScheduleToday %}
+{% from "../components/content-tile-large/macro.njk" import contentTileLarge %}
+{% from "../components/content-tile-small/macro.njk" import contentTileSmall %}
```
Comment (jonhindle) [inline server/views/pages/home-new.html:4]: These imports aren't used.

File: server/views/pages/home-new.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  {% if establishmentName != 'berwyn' %}
+    <div class="govuk-width-container home-navigation govuk-!-margin-top-3">
+      {{personalScheduleToday({ data: currentEvents, authReturnUrl: returnUrl }, userName if isSignedIn else '')}}
+    </div>
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-body home-content">
+    <div class="govuk-body home-content">
```
Comment (jonhindle) [inline server/views/pages/home-new.html:29]: unnecessary duplication of `<div class="govuk-body home-content">`

File: server/routes/__tests__/homepage.spec.js [hunk 3]
```diff
@@ -308,5 +305,153 @@ describe('GET /', () => {
...
+            location: 'New education',
+            timeString: '8:10AM to 11:25AM',
+            eventType: 'PRISON_ACT',
+            finished: false,
+            status: 'SCH',
+            paid: false,
+          },
+        ],
+        isTomorrow: false,
+      };
```
Comment (jonhindle) [inline server/routes/__tests__/homepage.spec.js:346]: Repeated setup. Maybe should all be in a describe too?
```
describe( 'New home page', () => {
  beforeEach(() => {
    const currentEvents = { ...stuff, stuff, stuff }
  })
  it('stuff', () => {
    stuff
  })
})
```

**Review by jonhindle — COMMENTED** [review]
Just tidying up kinda stuff I think.

**Review by leonk — APPROVED** [review]
LGTM

**Pull request merged**